src/main/kotlin/org/sertain/path/PathController.kt

```diff
@@ -0,0 +1,46 @@
+package org.sertain.path
+
+import jaci.pathfinder.Pathfinder
+import jaci.pathfinder.Trajectory
+import jaci.pathfinder.Waypoint
+import jaci.pathfinder.followers.EncoderFollower
+import jaci.pathfinder.modifiers.TankModifier
+
+object PathController {
+    val pathCoordinates = ArrayList<Triple<Double, Double, Double>>()
+
+    var dt = 0.05
+    var max_vel = 1.7
+    var max_accel = 2.0
+    var max_jerk = 60.0
+
+
+    fun addPath(xcoord: Double, ycoord: Double, radians: Double) {
+        pathCoordinates += Triple(xcoord, ycoord, radians)
+    }
+
+    fun configTrajectory(dt: Double, maximumVelocity: Double, maximumAcceleration: Double, maximumJerk: Double) {
+        this.dt = dt
+        this.max_vel = maximumVelocity
+        this.max_accel = maximumAcceleration
+        this.max_jerk = maximumJerk
+    }
+
+    fun generateEncoderFollowers() {
+        val points = ArrayList<Waypoint>()
+        for (coordset: Triple<Double, Double, Double> in pathCoordinates) {
+            points.add(Waypoint(coordset.first, coordset.second, coordset.third))
+        }
+        // Create Trajectory
```

Comment:
This comment doesn't mean anything, just fluff. Pretty obvious you're creating a trajectory here.